tools/mo/openvino/tools/mo/convert_impl.py

```diff
@@ -216,6 +216,13 @@ def raise_ie_not_found():
         log.error(e)
         raise_ie_not_found()
 
+    if ('compress_to_fp16' not in argv) or (argv.compress_to_fp16 is None):
+        argv.compress_to_fp16 = True
+        log.error("Be aware that --compress_to_fp16 argument was not specified. "
+                  "Generated IR will be compressed to fp16 by default. "
+                  "If you get lower accuracy, please consider disabling compression explicitly "
+                  "by adding argument --compress_to_fp16=False", extra={'is_warning': True})
```

Comment:
please take a look at this is implemented for tf_fe and ov api 2.0 info messages. I think we need to do it in the same way

Reply:
You mean this message?
https://github.com/openvinotoolkit/openvino/blob/ae1f9d1f9cb11e5ceb433c6fae9e2c60743b4d35/tools/mo/openvino/tools/mo/convert_impl.py#L457-L461

Reply:
took this as an example
https://github.com/openvinotoolkit/openvino/blob/6b74ce93635f7cf75994de0df611a3707d710274/tools/mo/openvino/tools/mo/utils/get_ov_update_message.py#L19-L33

Please review